settings_tab.py

```
@@ -0,0 +1,361 @@
+        except Exception as e:
+            print(f"Font reset error: {e}", file=sys.stderr)
+
+        try:
+            ui_theme.apply_ui_preferences(
+                self.root,
+                theme_mode=mode,
+                explicit_theme=theme_name or None,
+                font_family=family,
+                font_size=size,
+            )
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)